src/components/CreatePool/index.tsx

```diff
@@ -35,16 +35,14 @@ function CreatePool({ history }) {
   const [step, setStep] = useState<number>(1)
 
   const pair = usePair(token0, token1)
-  const pairExists = // used to detect new exchange
-    pair && JSBI.notEqual(pair.reserve0.raw, JSBI.BigInt(0)) && JSBI.notEqual(pair.reserve1.raw, JSBI.BigInt(0))
 
   useEffect(() => {
-    if (token0Address && token1Address && pair && !pairExists) {
+    if (token0Address && token1Address && !pair) {
```

Comment:
hm, what's the logic here? a bit hard to parse what the steps represent, a few comments here and there might helpful

Reply:
Yep replaced this with more clear steps for the views 

- one thing im considering is just linking straight to the regular add liquidity page. The only reason we keep the AddLiquidity component within this flow is so on the nav we can say "Create Pool" instead of 'Add Liquidity" on the last step (based on the route)

Reply:
Another thing is - at some point we discussed that this flow is probably best to let them pick tokens and then add - but should there ever be an option to just create the pool without adding? currently you cant do that in the UI

Reply:
> but should there ever be an option to just create the pool without adding? currently you cant do that in the UI

prob? kinda nice to cover the gas cost for someone else I guess if its like $5

Reply:
for new pools we could maybe popup a smaller option that says "create without liquidity" or something (on the add page)

Reply:
idk if the added confusion is worth helping the few people who want to create and not add though